examples/C++/HelloWorldExampleTCP/HelloWorldPublisher.h

```diff
@@ -34,7 +34,7 @@ class HelloWorldPublisher {
 	HelloWorldPublisher();
 	virtual ~HelloWorldPublisher();
 	//!Initialize
-	bool init();
+	bool init(const std::string &wan_ip, unsigned short port);
```

Comment:
TABS. Take the chance to change the whole file

Reply:
Fixed